app/views/ChooseProvider.js

```diff
@@ -232,9 +232,6 @@ class ChooseProviderScreen extends Component {
           title={languages.t('label.choose_provider_title')}
           onBackPress={this.backToMain.bind(this)}>
           <View style={styles.main}>
-            <Typography style={styles.headerTitle} use={'headline2'}>
-              {languages.t('label.authorities_title')}
```

Comment:
If this is the only place, remove the key from `en.json`

Reply:
Good catch.